infrastructure/infrastructure/src/main/java/alabno/utils/ConnUtils.java

```diff
@@ -9,11 +9,31 @@
  */
 public class ConnUtils {
 
+    @SuppressWarnings("unchecked")
     public static void sendAlert(WebSocket conn, String string) {
         JSONObject msgobj = new JSONObject();
         msgobj.put("type", "alert");
         msgobj.put("message", string);
         conn.send(msgobj.toJSONString());
     }
     
+    public enum Color {
+        RED, GREEN, YELLOW, BLACK;
```

Comment:
good point. We will improve this and change the enum to things like PROGRESS, SUCCESS, INFO, ERROR and let the client decide the style.